### PR TITLE
manits ilias ticket 36204

### PR DESCRIPTION
**Pull request by chlulei**
Link: https://mantis.ilias.de/view.php?id=36204

### CODE REVIEW

File: Modules/Course/classes/Objectives/class.ilCourseObjectivesTableGUI.php [hunk 1]
```diff
@@ -120,6 +120,14 @@ protected function fillRow(array $a_set): void
             $this->tpl->setVariable('LM_IMG', ilObject::_getIcon($data['obj_id'], "tiny", $data['type']));
             $this->tpl->setVariable('LM_ALT', $this->lng->txt('obj_' . $data['type']));
 
+            if ($data['online']) {
+                $this->tpl->setVariable('MAT_VAL_ONOFFLINE', $this->lng->txt('online'));
+                $this->tpl->setVariable('MAT_ONOFFLINE_CLASS', 'smallgreen');
```
Comment (smeyer-ilias) [inline Modules/Course/classes/Objectives/class.ilCourseObjectivesTableGUI.php]: Please avoid, to address the css classes in PHP files. 
Instead you could add two template blocks in tpl.crs_objectives_table_row.html and parse the one which is needed.

**Review by smeyer-ilias — CHANGES_REQUESTED** [review]
Thanks @chlulei 
See comment

File: Modules/Course/classes/Objectives/class.ilCourseObjectivesTableGUI.php (hunk 1 above)
Comment (chlulei) [reply Modules/Course/classes/Objectives/class.ilCourseObjectivesTableGUI.php]: The proposed changes have been implemented

**Review by smeyer-ilias — APPROVED** [review]
Thanks @chlulei

**Pull request merged**